src/query/functions-v2/src/scalars/math.rs

```diff
@@ -12,15 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::cmp::Ordering;
+use std::f64::consts::E;
 use std::f64::consts::PI;
+use std::marker::PhantomData;
 
+use common_base::containers::concat;
 use common_expression::types::number::NumberDomain;
+use common_expression::types::DataType;
 use common_expression::types::NumberType;
+use common_expression::types::StringType;
+use common_expression::with_number_mapped_type;
 use common_expression::FunctionProperty;
 use common_expression::FunctionRegistry;
 use common_expression::Value;
+use num_traits::AsPrimitive;
+use num_traits::Pow;
+use rand::Rng;
+use rand::SeedableRng;
 
 pub fn register(registry: &mut FunctionRegistry) {
+    let all_integer_types = &[
+        DataType::UInt8,
+        DataType::UInt16,
+        DataType::UInt32,
+        DataType::UInt64,
+        DataType::Int8,
+        DataType::Int16,
+        DataType::Int32,
+        DataType::Int64,
+    ];
+    let all_float_types = &[DataType::Float32, DataType::Float64];
+    let all_numerics_types = &concat(all_integer_types, all_float_types);
```

Comment:
duplicate with `all_numerics_types` in `arithmetic.rs`, maybe we can put them in `mod.rs`. They should be used elsewhere.